### DISCUSSION TITLE
Installation issues

### DISCUSSION THREAD

**Discussion by ahocken** [spectrochempy/spectrochempy#545] [Q&A] [answered]
Hello! I am having much difficulty installing spectrochempy with anaconda. It says the packages are installed, yet the build is not opening when called. I followed all of the instructions on the installation guide. Any suggestions would be appreciated! I've attached some of the code below:

(scpy_fix2) PS C:\Users\ChemeGrad2021> conda activate scpy_fix3
(scpy_fix3) PS C:\Users\ChemeGrad2021> mamba install spectrochempy
menuinst called from non-root env C:\Users\ChemeGrad2021\anaconda3\envs\scpy_fix3

                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (1.1.0) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████

Looking for: ['spectrochempy']

cantera/noarch                                     135.0 B @ 515.0 B/s  0.3s
cantera/win-64                                       3.2kB @  11.1kB/s  0.3s
spectrocat/win-64                                  138.0 B @ 418.0 B/s  0.3s
pkgs/msys2/win-64                                   39.8kB @ 101.7kB/s  0.1s
spectrocat/noarch                                    3.0kB @   6.8kB/s  0.5s
pkgs/msys2/noarch                                  111.0 B @ 158.0 B/s  0.3s
pkgs/r/noarch                                        1.3MB @ 991.4kB/s  1.0s
pkgs/main/noarch                                   818.6kB @ 532.3kB/s  0.9s
pkgs/r/win-64                                      742.8kB @ 414.0kB/s  0.5s
pkgs/main/win-64                                     4.5MB @   1.4MB/s  3.2s
conda-forge/noarch                                  10.8MB @   1.8MB/s  6.1s
conda-forge/win-64                                  18.6MB @   2.7MB/s  7.8s

Pinned packages:
  - python 3.8.*

Transaction

  Prefix: C:\Users\ChemeGrad2021\anaconda3\envs\scpy_fix3

  Updating specs:

   - spectrochempy
   - ca-certificates
   - certifi
   - openssl

  Package                                     Version  Build               Channel                 Size
---------------------------------------------------------------------------------------------------------
  Install:
---------------------------------------------------------------------------------------------------------

  + anyio                                       3.6.2  pyhd8ed1ab_0        conda-forge/noarch      85kB
  + appdirs                                     1.4.4  pyh9f0ad1d_0        conda-forge/noarch      13kB
  + argon2-cffi                                21.3.0  pyhd8ed1ab_0        conda-forge/noarch      16kB
  + argon2-cffi-bindings                       21.2.0  py38h91455d4_3      conda-forge/win-64      35kB
  + asttokens                                   2.2.1  pyhd8ed1ab_0        conda-forge/noarch      28kB
  + attrs                                      22.2.0  pyh71513ae_0        conda-forge/noarch      54kB
  + babel                                      2.11.0  pyhd8ed1ab_0        conda-forge/noarch       7MB
  + backcall                                    0.2.0  pyh9f0ad1d_0        conda-forge/noarch      14kB
  + backports                                     1.0  pyhd8ed1ab_3        conda-forge/noarch       6kB
  + backports.functools_lru_cache               1.6.4  pyhd8ed1ab_0        conda-forge/noarch       9kB
  + beautifulsoup4                             4.11.1  pyha770c72_0        conda-forge/noarch      98kB
  + bleach                                      5.0.1  pyhd8ed1ab_0        conda-forge/noarch     127kB
  + brotli                                      1.0.9  hcfcfb64_8          conda-forge/win-64      19kB
  + brotli-bin                                  1.0.9  hcfcfb64_8          conda-forge/win-64      23kB
  + brukeropusreader                            1.3.9  stable              spectrocat/noarch       19kB
  + comm                                        0.1.2  pyhd8ed1ab_0        conda-forge/noarch      11kB
  + contourpy                                   1.0.6  py38hb1fd069_0      conda-forge/win-64     179kB
  + cycler                                     0.11.0  pyhd8ed1ab_0        conda-forge/noarch      10kB
  + cython                                    0.29.33  py38hd3f51b4_0      conda-forge/win-64       2MB
  + debugpy                                     1.6.5  py38hd3f51b4_0      conda-forge/win-64       3MB
  + decorator                                   5.1.1  pyhd8ed1ab_0        conda-forge/noarch      12kB
  + defusedxml                                  0.7.1  pyhd8ed1ab_0        conda-forge/noarch      24kB
  + dill                                        0.3.6  pyhd8ed1ab_1        conda-forge/noarch      83kB
  + entrypoints                                   0.4  pyhd8ed1ab_0        conda-forge/noarch       9kB
  + executing                                   1.2.0  pyhd8ed1ab_0        conda-forge/noarch      25kB
  + flit-core                                   3.8.0  pyhd8ed1ab_0        conda-forge/noarch      46kB
  + fonttools                                  4.38.0  py38h91455d4_1      conda-forge/win-64       2MB
  + freetype                                   2.12.1  h546665d_1          conda-forge/win-64     497kB
  + gettext                                    0.21.1  h5728263_0          conda-forge/win-64       6MB
  + git                                        2.39.0  h57928b3_0          conda-forge/win-64      96MB
  + glib                                       2.74.1  h12be248_1          conda-forge/win-64     511kB
  + glib-tools                                 2.74.1  h12be248_1          conda-forge/win-64     190kB
  + gst-plugins-base                           1.21.3  h001b923_1          conda-forge/win-64       2MB
  + gstreamer                                  1.21.3  h6b5321d_1          conda-forge/win-64       2MB
  + icu                                          70.1  h0e60522_0          conda-forge/win-64      18MB
  + importlib-metadata                          6.0.0  pyha770c72_0        conda-forge/noarch      25kB
  + importlib_resources                        5.10.2  pyhd8ed1ab_0        conda-forge/noarch      30kB
  + intel-openmp                             2023.0.0  h57928b3_25922      conda-forge/win-64       2MB
  + ipykernel                                  6.20.1  pyh025b116_0        conda-forge/noarch     112kB
  + ipympl                                      0.9.2  pyhd8ed1ab_0        conda-forge/noarch     358kB
  + ipython                                     8.8.0  pyh08f2357_0        conda-forge/noarch     568kB
  + ipython_genutils                            0.2.0  py_1                conda-forge/noarch      22kB
  + ipywidgets                                  8.0.4  pyhd8ed1ab_0        conda-forge/noarch     111kB
  + jedi                                       0.18.2  pyhd8ed1ab_0        conda-forge/noarch     804kB
  + jinja2                                      3.1.2  pyhd8ed1ab_1        conda-forge/noarch     101kB
  + jpeg                                           9e  h8ffe710_2          conda-forge/win-64     375kB
  + json5                                       0.9.5  pyh9f0ad1d_0        conda-forge/noarch      21kB
  + jsonschema                                 4.17.3  pyhd8ed1ab_0        conda-forge/noarch      70kB
  + jupyter_client                              7.4.8  pyhd8ed1ab_0        conda-forge/noarch      99kB
  + jupyter_core                                5.1.3  py38haa244fe_0      conda-forge/win-64     106kB
  + jupyter_events                              0.5.0  pyhd8ed1ab_1        conda-forge/noarch      72kB
  + jupyter_server                              2.0.6  pyhd8ed1ab_0        conda-forge/noarch     304kB
  + jupyter_server_terminals                    0.4.3  pyhd8ed1ab_0        conda-forge/noarch      19kB
  + jupyterlab                                  3.5.2  pyhd8ed1ab_0        conda-forge/noarch       6MB
  + jupyterlab_pygments                         0.2.2  pyhd8ed1ab_0        conda-forge/noarch      17kB
  + jupyterlab_server                          2.18.0  pyhd8ed1ab_0        conda-forge/noarch      59kB
  + jupyterlab_widgets                          3.0.5  pyhd8ed1ab_0        conda-forge/noarch     173kB
  + kiwisolver                                  1.4.4  py38hb1fd069_1      conda-forge/win-64      60kB
  + lcms2                                        2.14  ha5c8aab_1          conda-forge/win-64     495kB
  + lerc                                        4.0.0  h63175ca_0          conda-forge/win-64     194kB
  + libblas                                     3.9.0  16_win64_mkl        conda-forge/win-64       6MB
  + libbrotlicommon                             1.0.9  hcfcfb64_8          conda-forge/win-64      69kB
  + libbrotlidec                                1.0.9  hcfcfb64_8          conda-forge/win-64      35kB
  + libbrotlienc                                1.0.9  hcfcfb64_8          conda-forge/win-64     742kB
  + libcblas                                    3.9.0  16_win64_mkl        conda-forge/win-64       6MB
  + libclang                                   15.0.6  default_h77d9078_0  conda-forge/win-64     147kB
  + libclang13                                 15.0.6  default_h77d9078_0  conda-forge/win-64      21MB
  + libdeflate                                   1.14  hcfcfb64_0          conda-forge/win-64      75kB
  + libglib                                    2.74.1  he8f3873_1          conda-forge/win-64       3MB
  + libhwloc                                    2.8.0  h039e092_1          conda-forge/win-64       3MB
  + libjpeg-turbo                               2.1.4  hcfcfb64_0          conda-forge/win-64       1MB
  + liblapack                                   3.9.0  16_win64_mkl        conda-forge/win-64       6MB
  + libogg                                      1.3.4  h8ffe710_1          conda-forge/win-64      35kB
  + libpng                                     1.6.39  h19919ed_0          conda-forge/win-64     344kB
  + libsodium                                  1.0.18  h8d14728_1          conda-forge/win-64     713kB
  + libtiff                                     4.5.0  hc4f729c_0          conda-forge/win-64       1MB
  + libvorbis                                   1.3.7  h0e60522_0          conda-forge/win-64     274kB
  + libwebp-base                                1.2.4  h8ffe710_0          conda-forge/win-64     335kB
  + libxcb                                       1.13  hcd874cb_1004       conda-forge/win-64       1MB
  + llvmlite                                   0.39.1  py38h19421c1_1      conda-forge/win-64      17MB
  + m2w64-gcc-libgfortran                       5.3.0  6                   conda-forge/win-64     351kB
  + m2w64-gcc-libs                              5.3.0  7                   conda-forge/win-64     532kB
  + m2w64-gcc-libs-core                         5.3.0  7                   conda-forge/win-64     219kB
  + m2w64-gmp                                   6.1.0  2                   conda-forge/win-64     744kB
  + m2w64-libwinpthread-git        5.0.0.4634.697f757  2                   conda-forge/win-64      32kB
  + markupsafe                                  2.1.1  py38h91455d4_2      conda-forge/win-64      26kB
  + matplotlib                                  3.6.2  py38haa244fe_0      conda-forge/win-64       8kB
  + matplotlib-base                             3.6.2  py38h528a6c7_0      conda-forge/win-64       8MB
  + matplotlib-inline                           0.1.6  pyhd8ed1ab_0        conda-forge/noarch      12kB
  + mistune                                     2.0.4  pyhd8ed1ab_0        conda-forge/noarch      69kB
  + mkl                                      2022.1.0  h6a75c08_874        conda-forge/win-64     192MB
  + msys2-conda-epoch                        20160418  1                   conda-forge/win-64       3kB
  + munkres                                     1.1.4  pyh9f0ad1d_0        conda-forge/noarch      12kB
  + nbclassic                                   0.4.8  pyhd8ed1ab_0        conda-forge/noarch       8MB
  + nbclient                                    0.7.2  pyhd8ed1ab_0        conda-forge/noarch      62kB
  + nbconvert                                   7.2.7  pyhd8ed1ab_0        conda-forge/noarch       8kB
  + nbconvert-core                              7.2.7  pyhd8ed1ab_0        conda-forge/noarch     199kB
  + nbconvert-pandoc                            7.2.7  pyhd8ed1ab_0        conda-forge/noarch       6kB
  + nbformat                                    5.7.1  pyhd8ed1ab_0        conda-forge/noarch     101kB
  + nest-asyncio                                1.5.6  pyhd8ed1ab_0        conda-forge/noarch      10kB
  + nodejs                                    18.12.1  h57928b3_0          conda-forge/win-64      26MB
  + notebook                                    6.5.2  pyha770c72_1        conda-forge/noarch     273kB
  + notebook-shim                               0.2.2  pyhd8ed1ab_0        conda-forge/noarch      15kB
  + numba                                      0.56.4  py38h3873db4_0      conda-forge/win-64       4MB
  + numpy                                      1.23.5  py38h90ce339_0      conda-forge/win-64       5MB
  + openjpeg                                    2.5.0  ha2aaf27_2          conda-forge/win-64     237kB
  + packaging                                    22.0  pyhd8ed1ab_0        conda-forge/noarch      40kB
  + pandoc                                     2.19.2  h57928b3_1          conda-forge/win-64      20MB
  + pandocfilters                               1.5.0  pyhd8ed1ab_0        conda-forge/noarch      12kB
  + parso                                       0.8.3  pyhd8ed1ab_0        conda-forge/noarch      71kB
  + pcre2                                       10.40  h17e33f8_0          conda-forge/win-64       2MB
  + pickleshare                                 0.7.5  py_1003             conda-forge/noarch       9kB
  + pillow                                      9.4.0  py38h409c3de_0      conda-forge/win-64      46MB
  + pint                                       0.20.1  pyhd8ed1ab_0        conda-forge/noarch     212kB
  + pkgutil-resolve-name                       1.3.10  pyhd8ed1ab_0        conda-forge/noarch       9kB
  + platformdirs                                2.6.2  pyhd8ed1ab_0        conda-forge/noarch      17kB
  + ply                                          3.11  py_1                conda-forge/noarch      45kB
  + pooch                                       1.6.0  pyhd8ed1ab_0        conda-forge/noarch      45kB
  + prometheus_client                          0.15.0  pyhd8ed1ab_0        conda-forge/noarch      51kB
  + prompt-toolkit                             3.0.36  pyha770c72_0        conda-forge/noarch     272kB
  + psutil                                      5.9.4  py38h91455d4_0      conda-forge/win-64     374kB
  + pthread-stubs                                 0.4  hcd874cb_1001       conda-forge/win-64       6kB
  + pthreads-win32                              2.9.1  hfa6e2cd_3          conda-forge/win-64     144kB
  + pure_eval                                   0.2.2  pyhd8ed1ab_0        conda-forge/noarch      15kB
  + pygments                                   2.14.0  pyhd8ed1ab_0        conda-forge/noarch     824kB
  + pyparsing                                   3.0.9  pyhd8ed1ab_0        conda-forge/noarch      81kB
  + pyqt                                       5.15.7  py38hd6c051e_2      conda-forge/win-64       5MB
  + pyqt5-sip                                 12.11.0  py38hd3f51b4_2      conda-forge/win-64      87kB
  + pyrsistent                                 0.19.3  py38h91455d4_0      conda-forge/win-64      92kB
  + python-dateutil                             2.8.2  pyhd8ed1ab_0        conda-forge/noarch     246kB
  + python-fastjsonschema                      2.16.2  pyhd8ed1ab_0        conda-forge/noarch     247kB
  + python-json-logger                          2.0.4  pyhd8ed1ab_0        conda-forge/noarch      13kB
  + pytz                                       2022.7  pyhd8ed1ab_0        conda-forge/noarch     186kB
  + pywin32                                       304  py38hd3f51b4_2      conda-forge/win-64      10MB
  + pywinpty                                   2.0.10  py38hd3f51b4_0      conda-forge/win-64     212kB
  + pyyaml                                        6.0  py38h91455d4_5      conda-forge/win-64     157kB
  + pyzmq                                      24.0.1  py38ha85f68a_1      conda-forge/win-64     473kB
  + qt-main                                    5.15.6  h9580fe5_5          conda-forge/win-64      62MB
  + quadprog                                   0.1.11  py38hbd9d945_1      conda-forge/win-64      84kB
  + quaternion                               2022.4.2  py38hbaf524b_1      conda-forge/win-64      82kB
  + scipy                                      1.10.0  py38h0f6ee2a_0      conda-forge/win-64      24MB
  + send2trash                                  1.8.0  pyhd8ed1ab_0        conda-forge/noarch      18kB
  + setuptools-scm                              7.1.0  pyhd8ed1ab_0        conda-forge/noarch      39kB
  + setuptools_scm                              7.1.0  hd8ed1ab_0          conda-forge/noarch       6kB
  + sip                                         6.7.5  py38hd3f51b4_0      conda-forge/win-64     485kB
  + six                                        1.16.0  pyh6c4a22f_0        conda-forge/noarch      14kB
  + sniffio                                     1.3.0  pyhd8ed1ab_0        conda-forge/noarch      14kB
  + soupsieve                             2.3.2.post1  pyhd8ed1ab_0        conda-forge/noarch      35kB
  + spectrochempy                              0.4.10  stable              spectrocat/noarch        2MB
  + stack_data                                  0.6.2  pyhd8ed1ab_0        conda-forge/noarch      26kB
  + tbb                                      2021.7.0  h91493d7_1          conda-forge/win-64     153kB
  + terminado                                  0.17.0  pyh08f2357_0        conda-forge/noarch      20kB
  + tinycss2                                    1.2.1  pyhd8ed1ab_0        conda-forge/noarch      23kB
  + toml                                       0.10.2  pyhd8ed1ab_0        conda-forge/noarch      18kB
  + tomli                                       2.0.1  pyhd8ed1ab_0        conda-forge/noarch      16kB
  + tornado                                       6.2  py38h91455d4_1      conda-forge/win-64     673kB
  + traitlets                                   5.8.1  pyhd8ed1ab_0        conda-forge/noarch      99kB
  + traittypes                                  0.2.1  pyh9f0ad1d_2        conda-forge/noarch      10kB
  + typing-extensions                           4.4.0  hd8ed1ab_0          conda-forge/noarch       9kB
  + typing_extensions                           4.4.0  pyha770c72_0        conda-forge/noarch      30kB
  + unicodedata2                               15.0.0  py38h91455d4_0      conda-forge/win-64     503kB
  + wcwidth                                     0.2.5  pyh9f0ad1d_2        conda-forge/noarch      34kB
  + webencodings                                0.5.1  py_1                conda-forge/noarch      12kB
  + websocket-client                            1.4.2  pyhd8ed1ab_0        conda-forge/noarch      44kB
  + widgetsnbextension                          4.0.5  pyhd8ed1ab_0        conda-forge/noarch     824kB
  + winpty                                      0.4.3  4                   conda-forge/win-64       1MB
  + xlrd                                        2.0.1  pyhd8ed1ab_3        conda-forge/noarch      94kB
  + xorg-libxau                                 1.0.9  hcd874cb_0          conda-forge/win-64      58kB
  + xorg-libxdmcp                               1.1.3  hcd874cb_0          conda-forge/win-64      68kB
  + yaml                                        0.2.5  h8ffe710_2          conda-forge/win-64      63kB
  + zeromq                                      4.3.4  h0e60522_1          conda-forge/win-64       9MB
  + zipp                                       3.11.0  pyhd8ed1ab_0        conda-forge/noarch      16kB

  Summary:

  Install: 172 packages

  Total download: 654MB

---------------------------------------------------------------------------------------------------------

Confirm changes: [Y/n] y
typing_extensions                                   30.0kB @ 133.8kB/s  0.2s
xlrd                                                94.1kB @ 389.4kB/s  0.2s
traitlets                                           98.5kB @ 391.1kB/s  0.3s
dill                                                82.7kB @ 317.9kB/s  0.3s
send2trash                                          17.5kB @  63.5kB/s  0.1s
attrs                                               54.1kB @ 190.2kB/s  0.0s
pint                                               212.4kB @ 724.2kB/s  0.3s
backcall                                            13.7kB @  45.1kB/s  0.0s
sniffio                                             14.4kB @  41.1kB/s  0.1s
six                                                 14.3kB @  35.8kB/s  0.1s
munkres                                             12.5kB @  30.2kB/s  0.1s
mistune                                             69.0kB @ 137.8kB/s  0.2s
webencodings                                        11.9kB @  22.1kB/s  0.1s
backports                                            6.0kB @  11.1kB/s  0.1s
python-fastjsonschema                              247.5kB @ 449.6kB/s  0.2s
widgetsnbextension                                 824.3kB @   1.5MB/s  0.3s
typing-extensions                                    8.6kB @  14.7kB/s  0.0s
ply                                                 44.8kB @  73.3kB/s  0.1s
traittypes                                          10.1kB @  16.4kB/s  0.1s
tinycss2                                            23.2kB @  37.3kB/s  0.1s
python-dateutil                                    246.0kB @ 388.5kB/s  0.1s
beautifulsoup4                                      98.3kB @ 152.0kB/s  0.1s
wcwidth                                             34.1kB @  52.0kB/s  0.0s
platformdirs                                        17.3kB @  26.2kB/s  0.0s
argon2-cffi-bindings                                35.4kB @  43.8kB/s  0.1s
lerc                                               194.4kB @ 231.5kB/s  0.2s
ipython                                            568.2kB @ 672.1kB/s  0.2s
psutil                                             373.9kB @ 414.2kB/s  0.3s
tornado                                            672.6kB @ 696.7kB/s  0.3s
kiwisolver                                          59.5kB @  61.7kB/s  0.1s
msys2-conda-epoch                                    3.2kB @   3.2kB/s  0.1s
sip                                                484.6kB @ 351.3kB/s  0.4s
libtiff                                              1.2MB @ 507.5kB/s  0.9s
cython                                               1.9MB @ 791.2kB/s  1.5s
libclang                                           146.6kB @  59.4kB/s  0.2s
libbrotlienc                                       741.8kB @ 241.3kB/s  0.7s
pyqt5-sip                                           86.6kB @  27.0kB/s  0.1s
m2w64-gcc-libs-core                                219.2kB @  65.1kB/s  0.2s
tbb                                                152.8kB @  43.7kB/s  0.1s
brotli                                              19.2kB @   5.4kB/s  0.1s
m2w64-gcc-libs                                     532.4kB @ 136.8kB/s  0.3s
gettext                                              5.6MB @   1.4MB/s  3.0s
pthread-stubs                                        6.4kB @   1.6kB/s  0.1s
libhwloc                                             3.1MB @ 650.3kB/s  2.3s
liblapack                                            5.9MB @ 749.6kB/s  3.9s
contourpy                                          179.1kB @  22.1kB/s  0.2s
quadprog                                            84.3kB @  10.2kB/s  0.2s
matplotlib                                           7.7kB @ 916.0 B/s  0.1s
libblas                                              5.9MB @ 701.8kB/s  4.5s
argon2-cffi                                         15.7kB @   1.9kB/s  0.1s
nbformat                                           101.0kB @  11.8kB/s  0.1s
ipykernel                                          111.5kB @  12.9kB/s  0.2s
numpy                                                5.2MB @ 599.7kB/s  3.9s
nbconvert-core                                     198.5kB @  22.8kB/s  0.2s
jupyterlab_server                                   59.1kB @   6.7kB/s  0.1s
packaging                                           40.5kB @   4.5kB/s  0.1s
jupyter_server                                     304.2kB @  34.1kB/s  0.3s
nest-asyncio                                         9.7kB @   1.1kB/s  0.1s
websocket-client                                    43.7kB @   4.8kB/s  0.1s
decorator                                           12.1kB @   1.3kB/s  0.1s
zipp                                                15.7kB @   1.7kB/s  0.1s
cycler                                              10.3kB @   1.1kB/s  0.1s
pure_eval                                           14.6kB @   1.6kB/s  0.1s
json5                                               20.9kB @   2.3kB/s  0.1s
soupsieve                                           34.7kB @   3.8kB/s  0.1s
comm                                                11.2kB @   1.2kB/s  0.1s
importlib_resources                                 30.4kB @   3.3kB/s  0.1s
anyio                                               85.2kB @   9.0kB/s  0.1s
setuptools-scm                                      39.1kB @   4.1kB/s  0.1s
prompt-toolkit                                     271.5kB @  27.4kB/s  0.4s
jedi                                               804.4kB @  80.0kB/s  0.7s
pyrsistent                                          91.5kB @   9.1kB/s  0.2s
pandoc                                              19.8MB @   1.9MB/s  9.6s
libjpeg-turbo                                        1.4MB @ 124.0kB/s  1.1s
winpty                                               1.2MB @ 105.3kB/s  1.1s
jupyterlab                                           5.5MB @ 487.7kB/s  2.6s
llvmlite                                            17.0MB @   1.5MB/s 10.5s
pywinpty                                           211.6kB @  18.2kB/s  0.3s
unicodedata2                                       503.2kB @  43.2kB/s  0.5s
libsodium                                          713.4kB @  61.1kB/s  0.3s
m2w64-libwinpthread-git                             31.9kB @   2.7kB/s  0.1s
openjpeg                                           237.1kB @  19.9kB/s  0.2s
glib-tools                                         189.8kB @  15.9kB/s  0.2s
glib                                               511.3kB @  41.9kB/s  0.3s
gst-plugins-base                                     2.0MB @ 154.1kB/s  1.1s
gstreamer                                            1.9MB @ 144.3kB/s  1.5s
pywin32                                              9.7MB @ 717.5kB/s  2.9s
libglib                                              3.3MB @ 238.0kB/s  2.2s
terminado                                           19.5kB @   1.4kB/s  0.1s
jupyter_server_terminals                            18.7kB @   1.3kB/s  0.1s
ipympl                                             357.7kB @  24.9kB/s  0.3s
notebook-shim                                       14.9kB @   1.0kB/s  0.1s
brukeropusreader                                    19.2kB @   1.3kB/s  0.3s
prometheus_client                                   51.0kB @   3.4kB/s  0.1s
pkgutil-resolve-name                                 8.7kB @ 582.0 B/s  0.1s
pygments                                           824.2kB @  52.7kB/s  0.7s
pyparsing                                           81.3kB @   5.1kB/s  0.2s
pandocfilters                                       11.6kB @ 727.0 B/s  0.1s
pytz                                               186.3kB @  11.5kB/s  0.2s
jupyterlab_pygments                                 17.4kB @   1.1kB/s  0.1s
matplotlib-base                                      7.9MB @ 481.1kB/s  2.9s
bleach                                             127.1kB @   7.7kB/s  0.2s
babel                                                7.0MB @ 359.0kB/s  3.2s
markupsafe                                          26.0kB @   1.3kB/s  0.1s
libwebp-base                                       335.4kB @  16.8kB/s  0.3s
icu                                                 17.9MB @ 886.2kB/s  9.0s
intel-openmp                                         2.2MB @ 105.3kB/s  1.1s
pcre2                                                2.0MB @  93.7kB/s  1.2s
freetype                                           497.4kB @  23.0kB/s  0.5s
libvorbis                                          273.7kB @  12.6kB/s  0.2s
m2w64-gmp                                          743.5kB @  34.1kB/s  0.5s
pyzmq                                              472.6kB @  21.4kB/s  0.2s
fonttools                                            1.8MB @  79.4kB/s  1.1s
libcblas                                             5.9MB @ 234.7kB/s  3.1s
jsonschema                                          70.5kB @   2.8kB/s  0.1s
jupyter_events                                      72.2kB @   2.9kB/s  0.1s
nbconvert-pandoc                                     6.1kB @ 240.0 B/s  0.1s
notebook                                           272.9kB @  10.7kB/s  0.2s
python-json-logger                                  13.2kB @ 513.0 B/s  0.1s
pickleshare                                          9.3kB @ 362.0 B/s  0.1s
executing                                           25.0kB @ 967.0 B/s  0.1s
appdirs                                             12.8kB @ 494.0 B/s  0.1s
importlib-metadata                                  24.6kB @ 945.0 B/s  0.1s
backports.functools_lru_cache                        9.0kB @ 346.0 B/s  0.1s
pyqt                                                 4.9MB @ 186.3kB/s  3.4s
libpng                                             343.9kB @  12.9kB/s  0.3s
scipy                                               23.6MB @ 725.4kB/s 19.1s
libogg                                              35.2kB @   1.1kB/s  0.1s
libbrotlidec                                        34.7kB @   1.1kB/s  0.1s
brotli-bin                                          22.9kB @ 699.0 B/s  0.1s
xorg-libxdmcp                                       67.9kB @   2.1kB/s  0.1s
libclang13                                          21.5MB @ 544.4kB/s 12.7s
jinja2                                             101.4kB @   2.6kB/s  0.1s
ipywidgets                                         111.4kB @   2.8kB/s  0.1s
spectrochempy                                        1.7MB @  40.7kB/s  0.9s
jupyterlab_widgets                                 172.8kB @   4.2kB/s  0.3s
defusedxml                                          24.1kB @ 586.0 B/s  0.1s
matplotlib-inline                                   12.3kB @ 298.0 B/s  0.1s
stack_data                                          26.2kB @ 636.0 B/s  0.1s
libdeflate                                          74.9kB @   1.8kB/s  0.1s
libbrotlicommon                                     69.1kB @   1.7kB/s  0.1s
pyyaml                                             156.5kB @   3.8kB/s  0.2s
m2w64-gcc-libgfortran                              350.7kB @   8.4kB/s  0.4s
nodejs                                              26.3MB @ 624.3kB/s 16.0s
jupyter_client                                      99.2kB @   2.3kB/s  0.2s
libxcb                                               1.3MB @  31.0kB/s  1.3s
ipython_genutils                                    21.6kB @ 498.0 B/s  0.1s
toml                                                18.4kB @ 425.0 B/s  0.1s
setuptools_scm                                       5.8kB @ 133.0 B/s  0.1s
yaml                                                63.3kB @   1.5kB/s  0.2s
qt-main                                             62.4MB @   1.4MB/s 32.4s
xorg-libxau                                         58.0kB @   1.3kB/s  0.2s
nbclient                                            61.8kB @   1.3kB/s  0.1s
entrypoints                                          9.2kB @ 199.0 B/s  0.1s
asttokens                                           27.8kB @ 600.0 B/s  0.1s
jpeg                                               374.7kB @   8.0kB/s  0.4s
lcms2                                              494.9kB @  10.5kB/s  0.4s
nbclassic                                            8.0MB @ 169.1kB/s  5.3s
flit-core                                           46.3kB @ 970.0 B/s  0.1s
numba                                                3.7MB @  74.5kB/s  2.3s
debugpy                                              2.8MB @  56.6kB/s  1.9s
tomli                                               15.9kB @ 320.0 B/s  0.1s
pthreads-win32                                     144.3kB @   2.9kB/s  0.1s
nbconvert                                            7.7kB @ 154.0 B/s  0.1s
jupyter_core                                       106.2kB @   2.1kB/s  0.1s
pooch                                               44.8kB @ 895.0 B/s  0.1s
quaternion                                          82.3kB @   1.6kB/s  0.2s
zeromq                                               9.4MB @ 185.6kB/s  6.8s
parso                                               71.0kB @   1.4kB/s  0.2s
pillow                                              46.0MB @ 749.8kB/s 28.5s
git                                                 96.2MB @   1.4MB/s 52.1s
mkl                                                191.6MB @   2.3MB/s 34.8s

Downloading and Extracting Packages

Preparing transaction: done
Verifying transaction: done
Executing transaction: done

C:\Users\ChemeGrad2021>if defined xml_catalog_files_libxslt (set XML_CATALOG_FILES= )  else (set XML_CATALOG_FILES= )

C:\Users\ChemeGrad2021>set xml_catalog_files_libxslt=

C:\Users\ChemeGrad2021>set xml_catalog_files_libxslt=

C:\Users\ChemeGrad2021>set XML_CATALOG_FILES=C:\Users\ChemeGrad2021\anaconda3\envs\scpy_fix3\etc\xml\catalog
(scpy_fix3) PS C:\Users\ChemeGrad2021> mamba install spectrochempy
menuinst called from non-root env C:\Users\ChemeGrad2021\anaconda3\envs\scpy_fix3

                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

pkgs/main/win-64                                              No change
cantera/noarch                                                No change
pkgs/main/noarch                                              No change
cantera/win-64                                                No change
pkgs/r/noarch                                                 No change
pkgs/msys2/win-64                                             No change
pkgs/msys2/noarch                                             No change
pkgs/r/win-64                                                 No change
spectrocat/win-64                                             No change
spectrocat/noarch                                             No change

Pinned packages:ch                                          Using cache
  - python 3.8.*

Transaction

  Prefix: C:\Users\ChemeGrad2021\anaconda3\envs\scpy_fix3

  All requested packages already installed

C:\Users\ChemeGrad2021>if defined xml_catalog_files_libxslt (set XML_CATALOG_FILES= )  else (set XML_CATALOG_FILES= )

C:\Users\ChemeGrad2021>set xml_catalog_files_libxslt=

C:\Users\ChemeGrad2021>set xml_catalog_files_libxslt=

C:\Users\ChemeGrad2021>set XML_CATALOG_FILES=C:\Users\ChemeGrad2021\anaconda3\envs\scpy_fix3\etc\xml\catalog
(scpy_fix3) PS C:\Users\ChemeGrad2021> from spectrochempy import*
At line:1 char:1
+ from spectrochempy import*
+ ~~~~
The 'from' keyword is not supported in this version of the language.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : ReservedKeywordNotAllowed

(scpy_fix3) PS C:\Users\ChemeGrad2021> spectrochempy
spectrochempy : The term 'spectrochempy' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ spectrochempy
+ ~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (spectrochempy:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by fernandezc** [comment, marked as answer]
Hi,

To run python program, you can not do it directly in the the command terminal.
First type the following command:

> activate scpy_fix3
> ipython

This open the python shell, where you can execute the import command

OR you do, instead of ipython

> Jupyter lab 

(as explained in the documentation) 

---- at least I think this can be your problem !

**Reply by fernandezc** [reply]
you can look here, for Jupyter lab:

https://www.spectrochempy.fr/latest/userguide/introduction/interface.html

**Reply by ahocken** [reply]
it seems to be working now! Thank you!
